cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -717,19 +728,11 @@ module {{bicepName .Name}} 'br/public:avm/res/app/container-app:0.8.0' = {
             value: {{bicepName .Name}}Identity.outputs.clientId
           }
           {{- end}}
-          {{- if (and .AzureEventHubs .AzureEventHubs.AuthUsingConnectionString) }}
+          {{- if (or (and .AzureEventHubs .AzureEventHubs.AuthUsingConnectionString) (and .AzureEventHubsForKafka .AzureEventHubsForKafka.AuthUsingConnectionString)) }}
           {
-            name: 'SPRING_CLOUD_AZURE_EVENTHUBS_CONNECTIONSTRING'
+            name: 'SPRING_CLOUD_AZURE_EVENTHUBS_CONNECTION_STRING'
```

Comment:
Please read this: https://github.com/azure-javaee/azure-dev/wiki/Knowledge-Sharing#2-sometimes-the-environment-variable-not-take-effect-in-spring-boot-application

> ![image](https://github.com/user-attachments/assets/fe17c26d-8acb-49e2-8eaf-4292a80fd6c7)